### DISCUSSION TITLE
A project that I would like to share

### DISCUSSION THREAD

**Discussion by Darin755** [copy/v86#662] [Show and tell]
Hello,
I have been working on on a project I call browser linux. My idea was to create a simple way of getting a linux comand prompt for various purposes. 
https://github.com/Darin755/browser-linux
Please let me know what you think.
![pfetch output](https://user-images.githubusercontent.com/54958995/170330580-8d5855e1-5f05-4132-b23b-e122cd9cc9a7.png)

**Comment by hello-smile6** [comment]
Nice! Got an issue heading your way...

**Reply by Darin755** [reply]
Did you see that I added neofetch?

**Reply by hello-smile6** [reply]
Yes

**Reply by Darin755** [reply]
Fell free to suggest anything else

**Reply by hello-smile6** [reply]
@Darin755 systemd?

**Comment by 621625** [comment]
In theory it is nice but it does not boot at all on my J7 Android phone.

I would like to test it if we can get it to run on mobile.

**Reply by hello-smile6** [reply]
Should work on mobile, does copy.sh/v86 work? Maybe wait a while.

**Reply by Darin755** [reply]
I am actually working on that.

**Reply by Darin755** [reply]
It takes a while on mobile. I found that Firefox on mobile is the only way it will boot.

**Comment by 621625** [comment]
I'm interested in a similar thing if I or you or we can get it to work on mobile.

Maybe use a smaller image.

-------- Original Message --------

**Reply by Darin755** [reply]
I have added the ability to use custom iso's via url parameters. You can try using the minimal iso with this url.
https://darin755.github.io/browser-linux?iso=minimal.iso

**Comment by 621625** [comment]
It's cool that you have added support for custom iso's.

It still doesn't work on my device, neither in Chrome or Brave.

Have a look at this site:

https://webassembly.sh

Wouldn't it be cool if it loaded fast like that?
-------- Original Message --------

**Reply by Darin755** [reply]
This looks like a cool project. I will have to look into it.

**Reply by hello-smile6** [reply]
@bitcoinmeetups webassembly.sh doesn't give you a real terminal. It runs binaries that are compiled to WebAssembly ahead of time.

**Comment by Darin755** [comment]
So I created a ton of documentation and added the ability to embed in other sites. My though was that it could be used to show off custom shell scripts.

**Reply by 621625** [reply]
I will try to test it. I'm looking for something like that. Do you have an e-mail address?

**Comment by 621625** [comment]
Nice loading screen but it still does not load in Brave browser, Android 7 which is what I'm targeting. Please let me know if you make a more generic demo that runs in mobile browsers.

**Comment by 621625** [comment]
Update: 

Buildroot v86 works in Brave mobile. And several of the other v86 emulators too. Not sure why this one doesn't.

**Reply by Darin755** [reply]
I just tested it in brave on e/os (android 11) and it worked out of the box. Can you double check that brave is not blocking it for some reason? If that is not the case you open a new issue in the repo.

**Reply by 621625** [reply]
I've got the minimal iso URL above to work now. Very cool. Is it possible to make it connect to the Internet without using the copy.sh rate limited tool?

**Reply by Darin755** [reply]
I wish
You may want to take a look at [this](https://github.com/copy/v86/issues/198) issue.

**Comment by 621625** [comment]
I know. But it looks good and it loads fast.

-------- Original Message --------

**Reply by Darin755** [reply]
What is the context for this message? Am I missing something?

**Reply by 621625** [reply]
Seems like the original message was missing somehow. It was there before. I was referring to a website which has a webassembly based Terminal.  webassembly.sh

**Reply by Darin755** [reply]
Oh, that makes more sense

**Comment by Darin755** [comment]
For anyone interested I am working on a version with almost instant load times: 

https://darin755.github.io/browser-linux?initial=minimal.bin.zst
